### ISSUE TITLE
No available video device on Ubuntu 21.04 and PopOS 21.04 

### ISSUE THREAD

**Issue by vdovhanych** [trezor/trezor-user-env#118]
Running`trezor-user-env` on `ubuntu` or `popos` on their latest versions results in the message `no available video device` when launching emulators. I haven't tried other distros.

This seems to be a problem with xhost and the connection to the server. 

Not a big deal for now as I can test it on other OS but we should try to find a solution for other users.

**Comment by tsusanka** [comment]
Weird. Did you try to log out and back in after installing xhost?

**Comment by vdovhanych** [comment]
Yeah, I did, and it even worked for me on the previous version, so I assume it's something to do with the change to wayland by default instead of xorg. I will do some digging once I have some spare time for it.

**Comment by karliatto** [comment]
I am experimenting the same issue in Pop!_OS 21.04

I am not sure if it might be some wrong configuration on my side, but it was working for me for some time in the same OS. Then it started giving error described in this issue.

**Comment by tsusanka** [comment]
Could you share what version of xhost is there?

**Comment by vdovhanych** [comment]
> Could you share what version of xhost is there?

`xhost 1.0.7` for me

**Comment by tsusanka** [comment]
Hm it seems I have problems also on macOS. Does it work for you on mac @vdovhanych?

**Comment by karliatto** [comment]
> Could you share what version of xhost is there?

I have the same as @vdovhanych  `xhost 1.0.7`

**Comment by vdovhanych** [comment]
> Hm it seems I have problems also on macOS. Does it work for you on mac @vdovhanych?

Looks like the x86 and arm ones are working fine for me on mac. Still seeing the same issue on popOS, was unable to fix it with anything I tried. 
<img width="1678" alt="Screenshot 2022-01-20 at 20 46 27" src="https://user-images.githubusercontent.com/45185420/150411326-9b86b85d-1cda-40ea-8cff-7d203e37466c.png">

**Comment by tsusanka** [comment]
@vdovhanych via docker or native nix?

**Comment by vdovhanych** [comment]
> 

This is docker setup

**Comment by tsusanka** [comment]
I am having problems with nix native, docker x86 and docker arm 🤣. Also "No available video device". Let's discuss it on our meeting but one more question what xQuartz version you have? I have 2.8.1 (xorg-server 1.20.11).

**Comment by vdovhanych** [comment]
> 

Ok :D, i have the same one as you.

**Comment by tsusanka** [comment]
So I have restarted my laptop and it works 😆.

**Comment by vdovhanych** [comment]
Figured this out, DISPLAY is set to :1 by default. The output of `echo $DISPLAY` gives you `:1` solution is to either edit the compose file or change the display env to  `- DISPLAY=:1` at which point it will work.

**Comment by tsusanka** [comment]
Did you have more than one display plugged in? 🤔

**Comment by vdovhanych** [comment]
Nope, it's on my laptop with only built-in display

**Comment by tsusanka** [comment]
On my machine:
```
echo $DISPLAY
/private/tmp/com.apple.launchd.M4CXQuwjne/org.xquartz:0
```

**Comment by karliatto** [comment]
So I tried doing `- DISPLAY=:1` in the `docker-compose` and it works for me!

**Comment by tsusanka** [comment]
As discussed on our meeting: can we use the `$DISPLAY` variable directly? 🤔 

That means replacing this with `$DISPLAY`?

https://github.com/trezor/trezor-user-env/blob/487d2d3647722d7f7fa51c1c24c9f1c6f8c3bd3c/docker/compose.yml#L8

**Comment by tsusanka** [comment]
It works, @vdovhanych will file a PR 🚀 .
